Examples/AblyAssetTracking.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -14,9 +14,9 @@
         "package": "Amplify",
         "repositoryURL": "https://github.com/aws-amplify/amplify-ios",
         "state": {
-          "branch": "main",
```

Comment:
Do these dependency changes need to be in this commit?

Reply:
it's automatically managed by Xcode when I'm opening an Example app. 
I probably shouldn't push it but I'll leave it here.